### ISSUE TITLE
Work in progress: timelapse support

### ISSUE THREAD

**Issue by lazigu** [BiAPoL/napari-clusters-plotter#58]
Hi @Cryaaa, I need your help with one thing, maybe you can pinpoint the issue faster. The issue might have something to do with the code for the visualization of the cluster image, but it also might be somewhere deeper.
After seeing that video yesterday I was playing around with timelapses on [this branch](https://github.com/BiAPoL/napari-clusters-plotter/tree/try-timelapse-support), and so far what works:

- Getting measurements for a timelapse. It loops through the whole timelapse and gets reg props data for each time point and in the end concatenates all _(works for now just for area & intensity measurements, not with neighborhood data)._ There is also a "timepoint" column, which we might rename to "frame" to be consistent with Trackmate results.
- Then you can plot timepoint against the label, and nicely see if visualization works correctly (6th cell divides at 6th timepoint, so we get one more label, and it's good for looking for bugs):
![image](https://user-images.githubusercontent.com/52177660/157856597-df475e58-a1f0-419c-b641-30f0abc53e0c.png)

It works well for 2-6 labels, but not 1st and 7th. This is what it displays if the 1st label is selected with a picker tool:
![image](https://user-images.githubusercontent.com/52177660/157856744-84b37e74-1ec8-4805-8a2f-2c79a267a64e.png)

I was hoping we could discuss this next week in one of the meetings when we are both there, so I am just putting this here so if you wanted you could already take a look. Tell me if you want to send me this segmented dataset, it's 4D but I am using just 16 timepoints, so it's great to try out stuff because it's not computationally demanding.

**Comment by Cryaaa** [comment]
Hey @lazigu!
Definitely send the dataset (we transfer link or whatever you use usually would be best) so I can try this myself. I will look into this a bit, maybe we can discuss this issue on Tuesday next week in detail?

**Comment by lazigu** [comment]
@Cryaaa good, Tuesday fits! I sent to you the data.

**Comment by Cryaaa** [comment]
@lazigu just a small update. The problem seems to lie somewhere deep (not just with the generation of the cluster image). What happens is that the label column is displayed properly in the table but the connection to the labels in the image is wrong. For example the cell in the image with label = 1 gets given label 2 from the table and this shift is throughout the whole table. I'll see if I can track down where this issue is caused or we'll brainstorm tomorrow if I can't find anything today..

**Comment by lazigu** [comment]
Forgot to mention, for the fix to work napari skimage regioprops newest version should be installed im the environment.

**Comment by Cryaaa** [comment]
@lazigu that might be it then! I'll give it a try where the Roberts fix is and that might be it already. I'm also very sure that exactly that bug Robert fixed is causing the issues at the start of this discussion..

**Comment by lazigu** [comment]
@Cryaaa I don’t think so that it is related to this issue :/ It fixes what is displayed in napari status bar, but actually nowhere else we are using the index column.

**Comment by Cryaaa** [comment]
@lazigu I think you're right there. I found that if there is a background label 0 with properties of the background that the problem we are having might be solved. The thing is then we always have the background in the table (even multiple backgrounds for timelapse images). But maybe Robert @haesleinhuepf knows a way to remove or ignore the background labels in the napari skimage regionprops table? I think we should discuss the rest in person tomorrow though, it will make explanations much easier.

**Comment by Cryaaa** [comment]
@lazigu, nevermind, I found the problem and it lied within the plotter widget. The next commit to this branch should fix it!

**Comment by lazigu** [comment]
To do:

- Discuss how to handle background label/row in layer.properties/reg props table #56.
- Should it be standardized that uploaded reg props from the file need to contain timepoint/frame column?
- Settle on the name for the timepoint/frame column.
- Saving manual clustering results in the reg props table.
- Implement that the size of data points in the plot changes depending on how many data points are displayed.

@Cryaaa feel free to add points to this mini list! 
And could you repeat what was still wrong with napari status bar? I tried it out now but didn't notice any bug when switching to different time points.
